libcontainer/nsenter/nsexec.c

```diff
@@ -1223,8 +1224,14 @@ void nsexec(void)
 			 * was broken, so we'll just do it the long way anyway.
 			 */
 			write_log(DEBUG, "unshare remaining namespace (except cgroupns)");
-			if (unshare(config.cloneflags & ~CLONE_NEWCGROUP) < 0)
-				bail("failed to unshare remaining namespaces (except cgroupns)");
+			for (; retries > 0; retries--) {
```

Comment:
Please add a comment line to explain why this retry loop is needed

Reply:
Done. Thank you for your suggestion.